app/models/user.rb

```diff
@@ -645,4 +646,10 @@ def update_rate_limit
   rescue RateLimitChecker::LimitReached => e
     errors.add(:base, "User could not be saved. #{e.message}")
   end
+
+  def password_matches_confirmation
+    return true if password == password_confirmation
```

Comment:
can we use secure compare here? 

https://github.com/forem/forem/blob/44b8d0894d00ceb78aa30a35c6bfe62536e3c90b/app/controllers/api/v0/api_controller.rb#L71-L73

Reply:
`secure_compare` prevents timing attacks when a user-provided value is hashed and compared to a stored hash. This is not the case here, both values are user provided and we compare them against each other, not against something stored in the DB.